opennms-webapp-rest/src/main/java/org/opennms/web/rest/v1/EventRestService.java

```diff
@@ -258,6 +263,14 @@ private void processEventAck(final SecurityContext securityContext, final OnmsEv
         m_eventDao.save(event);
     }
 
+    @POST
+    @Consumes({MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON, MediaType.APPLICATION_ATOM_XML})
+    @Transactional
+    public Response publishEvent(final org.opennms.netmgt.xml.event.Event event) {
+        m_eventForwarder.sendNow(event);
```

Comment:
sendNow will only throw an exception if we get a RejectedExecutionException from the thread pool, in which case the default behavior of a 500 is probably suitable.

Did you have something specific in mind?